sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTraceExporterEventSource.cs

```diff
@@ -71,5 +79,40 @@ private static string GetMessage(object value)
         {
             return value is Exception exception ? exception.ToInvariantString() : value.ToString();
         }
+
+#if DEBUG
+        public class AzureMonitorTraceExporterEventListener : EventListener
+        {
+            private readonly List<EventSource> eventSources = new List<EventSource>();
+
+            public override void Dispose()
+            {
+                foreach (EventSource eventSource in this.eventSources)
+                {
+                    this.DisableEvents(eventSource);
+                }
+
+                base.Dispose();
+                GC.SuppressFinalize(this);
+            }
+
+            protected override void OnEventSourceCreated(EventSource eventSource)
+            {
+                if (eventSource?.Name.StartsWith("OpenTelemetry", StringComparison.OrdinalIgnoreCase) == true)
```

Comment:
Is it intentional to listen for all OpenTelemetry*** ? Or we just need to look for the ones from this exporter alone?

Reply:
Good catch. My intention was to capture only events from this exporter. Renamed the eventsource name to start with Microsoft.OpenTelemetry*